cluster/store/store.go

```diff
@@ -570,7 +570,7 @@ func (st *Store) Execute(req *api.ApplyRequest) (uint64, error) {
 	}
 	if req.Type == api.ApplyRequest_TYPE_ADD_CLASS {
 		if other := st.FindSimilarClass(req.Class); other == req.Class {
-			return 0, errClassExists
+			return 0, fmt.Errorf("class name %s %w", req.Class, errClassExists)
```

Comment:
**suggestion:** let's put it as the expected msg, because i think errClassExists=`class already exists`. 

also IIRC i think there is a tests the `acceptance_with_go_client` could fail because expecting specific msg